content/applications/marketing/marketing_automation/target_audience.rst

```diff
@@ -1,123 +1,123 @@
 ==================
-Target an audience
+Audience targeting
 ==================
 
-Delivering marketing campaigns to the right audience is paramount when trying to grow a business.
-The Odoo *Marketing Automation* application helps marketers to do just that by providing detailed
-filtering tools, which can be as simple (or as complex) as necessary, to reach the right customers
-at the right time.
+The *Target* and *Filter* section of the campaign form, also referred to as the domain, contains the
+fields used to define the target audience for the campaign's reach (i.e., the unique contact records
+in the database).
```

Comment:
- would use guis here since we're talking about fields on the UI, otherwise uncapitalized italics since they're not proper nouns
- wording: these are fields on a form, not sections as would be indicated by headings
- it's also know clear what exactly the *domain* is referring to: one field in particular or both of them together?
```suggestion
The :guilabel:`Target` and :guilabel:`Filter` fields on the campaign form, also referred to as the *domain*, contain the
parameters used to define the target audience for the campaign's reach (i.e., the unique contact records
in the database, and imported list, etc.).
```